src/app/loadout/ModPicker.tsx

```diff
@@ -249,28 +215,61 @@ function mapStateToProps() {
  * chosen too many mods. It also can be filtered down to a specific set of mods
  * using plugCategoryHashWhitelist.
  */
-function ModPicker({ plugSets, classType, lockedMods, initialQuery, onAccept, onClose }: Props) {
-  // Partition the locked (selected) mods into ones that will be shown in this
-  // picker (based on plugCategoryHashWhitelist) and ones that will not. The
-  // ones that won't (hidden mods) are still part of the locked mods set and
-  // shouldn't be wiped out when the selection of visible mods changes!
-  const [_visibleSelectedMods, hiddenSelectedMods] = useMemo(
```

Comment:
`visibleSelectedMods` wasn't used so I switched this to just a filter and moved it into the memo below.